torch/nn/init.py

```diff
@@ -87,6 +87,26 @@ def normal(tensor, mean=0, std=1):
     return tensor.normal_(mean, std)
 
 
+def random_gamma(tensor, alpha=1, beta=1):
```

Comment:
Is that actually used as an initialization method for weights?

Reply:
No, removed.